Makefile

```diff
@@ -51,16 +51,17 @@ mvn:
 	@echo ===================================
 	make lib-mvn
 	./mvnw clean package
-	ln -sfv target/check_kafka-*.jar check_kafka.jar
+	ln -sfv target/check_kafka-*.jar check_kafka_broker.jar
```

Comment:
is this just making the symlink more specific?

Reply:
yes, I just thought there would be more kafka check in the future. check_kafka is too vague and might mislead ppl among all other checks.